source/renderer/app/i18n/locales/en-US.json

```diff
@@ -1,816 +1,816 @@
-  "wallet.transaction.filter.incoming": "Ada received",
-  "wallet.transaction.filter.last30Days": "Last 30 days",
-  "wallet.transaction.filter.last7Days": "Last 7 days",
-  "wallet.transaction.filter.last90Days": "Last 90 days",
-  "wallet.transaction.filter.outgoing": "Ada sent",
-  "wallet.transaction.filter.resetFilter": "Reset Filter",
-  "wallet.transaction.filter.resultInfo": "<b>{filtered}</b> out of <b>{total}</b> transactions match your filter.",
-  "wallet.transaction.filter.selectTimeRange": "Select time range",
-  "wallet.transaction.filter.thisYear": "This year",
-  "wallet.transaction.noInputAddressesLabel": "No addresses",
-  "wallet.transaction.pending.cancelPendingTxnNote": "This transaction has been pending for a long time. To release the funds used by this transaction, you can try canceling it.",
-  "wallet.transaction.pending.cancelTransactionButton": "Cancel pending transaction",
-  "wallet.transaction.pending.supportArticleLink": "Why should I cancel this transaction?",
-  "wallet.transaction.pending.supportArticleUrl": "https://iohk.zendesk.com/hc/en-us/articles/360038113814",
-  "wallet.transaction.received": "{currency} received",
-  "wallet.transaction.sent": "{currency} sent",
-  "wallet.transaction.state.confirmed": "Transaction confirmed",
-  "wallet.transaction.state.pending": "Transaction pending",
-  "wallet.transaction.transactionAmount": "Transaction amount",
-  "wallet.transaction.transactionId": "Transaction ID",
-  "wallet.transaction.type": "{currency} transaction",
-  "wallet.transaction.type.card": "Card Payment",
-  "wallet.transaction.type.exchange": "Exchange",
-  "wallet.transaction.unresolvedInputAddressesAdditionalLabel": "to see these addresses.",
-  "wallet.transaction.unresolvedInputAddressesLinkLabel": "Open this transaction in Cardano explorer",
-  "wallet.transactions.no.transactions": "No transactions",
-  "wallet.transferFunds.dialog1.continueLabel": "Continue",
-  "wallet.transferFunds.dialog1.sourceWallet": "From Balance wallet",
-  "wallet.transferFunds.dialog1.targetWallet": "To Rewards wallet",
-  "wallet.transferFunds.dialog1.title": "Transfer funds from the Balance wallet",
-  "wallet.transferFunds.dialog2.label.amount": "Amount",
-  "wallet.transferFunds.dialog2.label.buttonLabel": "Transfer funds",
-  "wallet.transferFunds.dialog2.label.description": "Confirm transfer from the {sourceWalletName} wallet to the {targetWalletName} wallet.",
-  "wallet.transferFunds.dialog2.label.fees": "Fees",
-  "wallet.transferFunds.dialog2.label.to": "To",
-  "wallet.transferFunds.dialog2.label.total": "Total",
-  "wallet.transferFunds.dialog2.passphraseFieldPlaceholder": "Type your spending password",
-  "wallet.transferFunds.dialog2.passphraseLabel": "Spending password",
-  "wallet.transferFunds.dialog2.title": "Transfer funds from the Balance wallet"
-}
\ No newline at end of file
+	"api.errors.ApiMethodNotYetImplementedError": "This API method is not yet implemented.",
```

Comment:
@nikolaglumac, is this acceptable or I need to find a way to fix this? This was automatically done by BabelEdit with project creation.

Reply:
There is no settings to avoid this so we can fix this when the work is done, before we merge the PR.

Reply:
@darko-mijic I have fixed this manually.